src/schema/yaml-1.1/timestamp.ts

```diff
@@ -47,7 +47,7 @@ function stringifySexagesimal(node: Scalar) {
   return (
     sign +
     parts
-      .map(n => (n < 10 ? '0' + String(n) : String(n)))
+      .map(n => String(n).padStart(2, '0')) //(n < 10 ? '0' + String(n) : String(n)))
```

Comment:
```suggestion
      .map(n => String(n).padStart(2, '0'))
```

Reply:
Oh, haha, I didn't mean to leave that in there, but sure :)